pom.xml

```diff
@@ -0,0 +1,241 @@
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>edu.ucla.library</groupId>
+  <artifactId>services-utilities</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>services-utilities</name>
+  <description>Simple utilities for UCLA LIbrary Services Team projects</description>
+  
+  <licenses>
+    <license>
+      <name>The 3-Clause BSD License</name>
+      <url>https://opensource.org/licenses/BSD-3-clause</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>DRickard</id>
+      <name>David Rickard</name>
+      <email>drickard1967@library.ucla.edu</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+    <developer>
+      <id>ksclarke</id>
+      <name>Kevin S. Clarke</name>
+      <email>ksclarke@ksclarke.io</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/New_York</timezone>
+    </developer>
+    <developer>
+      <id>cachemeoutside</id>
+      <name>Anthony Vuong</name>
+      <email>avuong@cachemeoutside.io</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+    <developer>
+      <id>markmatney</id>
+      <name>Mark A. Matney, Jr.</name>
+      <email>mmatney@library.ucla.edu</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+  </developers>
+
+  <properties>
+    <maven-compiler.version>3.8.1</maven-compiler.version>
+    <surefire.version>3.0.0-M5</surefire.version>
+    <codacy.version>1.2.0</codacy.version>
+    <junit.version>4.13</junit.version>
+    <system-rules.version>1.19.0</system-rules.version>
+    <system-lambda.version>1.1.0</system-lambda.version>
+  </properties>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+    <testResources>
+      <testResource>
+        <directory>src/test/resources</directory>
+        <filtering>true</filtering>
+      </testResource>
+    </testResources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler.version}</version>
+        <configuration>
+          <release>11</release>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+	<configuration>
```

Comment:
Looks like indentation is off.

Reply:
Hmmm... indentation looks fine in both my editor and in the browser: https://github.com/UCLALibrary/services_utilities/blob/SERV-22/pom.xml

Reply:
![Screenshot from 2020-09-02 12-31-23](https://user-images.githubusercontent.com/106238/92010953-681dc180-ed18-11ea-819c-c258dfac493b.png)

Hmm, this is what it looks like in my browser... `configuration` is at the same level as plugin, but it's a child element of plugin.

Reply:
Hmmm... checked in a different editor and saw the funky indentation you highlighted.

Reply:
There was the same for version in the codacy plugin. Could be a spaces vs. tabs issue if you're not seeing it in your other editor?